cities-temperatures/src/main/java/City.java

```diff
@@ -0,0 +1,15 @@
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+@lombok.EqualsAndHashCode(of = {"name", "temperature"})
```

Comment:
Yes, but the of part is redundant - by default, it generates an equals and a hash code based on all the fields

Reply:
plus, please simplify it to `@EqualsAndHashCode`